Macos-14 builds fail if output_dir does not already exist AND testing is enabled

### Description

Building for Macos14 (in github actions = arm64) fails when attempting to move the wheel after testing if the output directory does not already exist.

Contrary to the documentation `pathlib.Path.unlink()` throws a `NotADirectoryError` in this case, and apparently only on this platform. (I will raise an Issue in cpython for that as well - at least to update the docs ...)

Compare:
- Failed run: https://github.com/MusicalNinjaDad/FizzBuzz/actions/runs/9351587993/job/25744497746
```
============================== 11 passed in 0.05s ==============================
  Traceback (most recent call last):
    File "/Users/runner/work/_temp/cibw/bin/cibuildwheel", line 8, in <module>
      sys.exit(main())
               ^^^^^^
    File "/Users/runner/work/_temp/cibw/lib/python3.12/site-packages/cibuildwheel/__main__.py", line 143, in main
      build_in_directory(args)
    File "/Users/runner/work/_temp/cibw/lib/python3.12/site-packages/cibuildwheel/__main__.py", line 315, in build_in_directory
      platform_module.build(options, tmp_path)
    File "/Users/runner/work/_temp/cibw/lib/python3.12/site-packages/cibuildwheel/macos.py", line 643, in build
      (build_options.output_dir / repaired_wheel.name).unlink()
    File "/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/pathlib.py", line 1342, in unlink
      os.unlink(self)
  NotADirectoryError: [Errno 20] Not a directory: '/Users/runner/work/FizzBuzz/FizzBuzz/wheelhouse/fizzbuzz-2.1.0-cp39-cp39-macosx_11_0_arm64.whl'
  Error: Process completed with exit code 1.
```
- Previously passed (no tests): https://github.com/MusicalNinjaDad/FizzBuzz/actions/runs/9108520076/job/25039459108
- Subsequently passed with workaround (added `&& mkdir wheelhouse` to `tool.cibuildwheel.macos.before-all`): https://github.com/MusicalNinjaDad/FizzBuzz/actions/runs/9354284188/job/25746806904

### Build log

https://github.com/MusicalNinjaDad/FizzBuzz/actions/runs/9351587993/job/25744497746

### CI config

https://github.com/MusicalNinjaDad/FizzBuzz/tree/pr48

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -641,8 +641,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                with contextlib.suppress(FileNotFoundError):
-                    (build_options.output_dir / repaired_wheel.name).unlink()
+                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -640,7 +640,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
+                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -1,6 +1,5 @@
 from __future__ import annotations
 
-import contextlib
 import functools
 import os
 import platform

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -36,6 +36,7 @@ from .util import (
     get_build_verbosity_extra_flags,
     get_pip_version,
     install_certifi_script,
+    move_file,
     prepare_command,
     read_python_configs,
     shell,
@@ -640,16 +641,10 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                output_dir = build_options.output_dir.resolve()
-                output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
-                output_wheel.unlink(missing_ok=True)
-
-                # shutil.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
-                output_dir.mkdir(parents=True, exist_ok=True)
-
-                # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                # explicit str() needed for Python 3.8
-                shutil.move(str(repaired_wheel), str(output_wheel))
+                output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
+                moved_wheel = move_file(repaired_wheel, output_wheel)
+                if moved_wheel != output_wheel.resolve():
+                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -640,10 +640,16 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
+                output_dir = build_options.output_dir.resolve()
+                output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
+                output_wheel.unlink(missing_ok=True)
 
-                shutil.move(str(repaired_wheel), build_options.output_dir)
-                built_wheels.append(build_options.output_dir / repaired_wheel.name)
+                # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
+                output_dir.mkdir(parents=True, exist_ok=True)
+                
+                # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
+                shutil.move(repaired_wheel, output_wheel)
+                built_wheels.append(output_wheel)
 
             # clean up
             shutil.rmtree(identifier_tmp_dir)

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -644,13 +644,13 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
                 output_wheel.unlink(missing_ok=True)
 
-                # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
+                # shutil.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
                 output_dir.mkdir(parents=True, exist_ok=True)
 
-                # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
+                # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
                 shutil.move(
                     str(repaired_wheel), str(output_wheel)
-                )  # explicit str() needed for mypy3.8
+                )  # explicit str() needed for Python 3.8 - can change to Paths when we drop 3.8 support
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -648,9 +648,8 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(
-                    str(repaired_wheel), str(output_wheel)
-                )  # explicit str() needed for Python 3.8 - can change to Paths when we drop 3.8 support
+                # explicit str() needed for Python 3.8
+                shutil.move(str(repaired_wheel), str(output_wheel))
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -646,7 +646,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
                 # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
                 output_dir.mkdir(parents=True, exist_ok=True)
-                
+
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
                 shutil.move(repaired_wheel, output_wheel)
                 built_wheels.append(output_wheel)

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -648,7 +648,7 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(repaired_wheel, output_wheel)
+                shutil.move(str(repaired_wheel), str(output_wheel)) # explicit str() needed for mypy3.8
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -648,7 +648,9 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(str(repaired_wheel), str(output_wheel)) # explicit str() needed for mypy3.8
+                shutil.move(
+                    str(repaired_wheel), str(output_wheel)
+                )  # explicit str() needed for mypy3.8
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/macos.py
+++ b/cibuildwheel/macos.py
@@ -644,7 +644,9 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
                 moved_wheel = move_file(repaired_wheel, output_wheel)
                 if moved_wheel != output_wheel.resolve():
-                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
+                    log.warning(
+                        "{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}"
+                    )
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/pyodide.py
+++ b/cibuildwheel/pyodide.py
@@ -395,8 +395,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                with contextlib.suppress(FileNotFoundError):
-                    (build_options.output_dir / repaired_wheel.name).unlink()
+                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/pyodide.py
+++ b/cibuildwheel/pyodide.py
@@ -1,6 +1,5 @@
 from __future__ import annotations
 
-import contextlib
 import os
 import shutil
 import sys

--- a/cibuildwheel/pyodide.py
+++ b/cibuildwheel/pyodide.py
@@ -394,7 +394,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
+                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/pyodide.py
+++ b/cibuildwheel/pyodide.py
@@ -26,6 +26,7 @@ from .util import (
     extract_zip,
     find_compatible_wheel,
     get_pip_version,
+    move_file,
     prepare_command,
     read_python_configs,
     shell,
@@ -394,9 +395,11 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (overwrite existing)
             if compatible_wheel is None:
-                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
+                output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
+                moved_wheel = move_file(repaired_wheel, output_wheel)
+                if moved_wheel != output_wheel.resolve():
+                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
+                built_wheels.append(output_wheel)
 
-                shutil.move(str(repaired_wheel), build_options.output_dir)
-                built_wheels.append(build_options.output_dir / repaired_wheel.name)
     finally:
         pass

--- a/cibuildwheel/pyodide.py
+++ b/cibuildwheel/pyodide.py
@@ -398,7 +398,9 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
                 moved_wheel = move_file(repaired_wheel, output_wheel)
                 if moved_wheel != output_wheel.resolve():
-                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
+                    log.warning(
+                        "{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}"
+                    )
                 built_wheels.append(output_wheel)
 
     finally:

--- a/cibuildwheel/util.py
+++ b/cibuildwheel/util.py
@@ -382,7 +382,8 @@ def move_file(src_file: Path, dst_file: Path) -> Path:
 
     if dst_file.is_dir():
         # Cannot overwrite a directory with a file
-        raise IsADirectoryError
+        msg = "dst_file must be a valid target filename, not an existing directory."
+        raise IsADirectoryError(msg)
     dst_file.unlink(missing_ok=True)
     dst_file.parent.mkdir(parents=True, exist_ok=True)
 

--- a/cibuildwheel/util.py
+++ b/cibuildwheel/util.py
@@ -6,6 +6,7 @@ import itertools
 import os
 import re
 import shlex
+import shutil
 import ssl
 import subprocess
 import sys
@@ -358,6 +359,40 @@ def extract_tar(tar_src: Path, dest: Path) -> None:
         tar_.extraction_filter = getattr(tarfile, "tar_filter", (lambda member, _: member))
         tar_.extractall(dest)
 
+
+def move_file(src_file: Path, dst_file: Path) -> Path:
+    """Moves a file safely while avoiding potential semantic confusion:
+     
+    1. `dst_file` must point to the target filename, not a directory
+    2. `dst_file` will be overwritten if it already exists
+    3. any missing parent directories will be created
+
+    Returns the fully resolved Path of the resulting file.
+
+    Raises:
+        NotADirectoryError: If any part of the intermediate path to `dst_file` is an existing file
+        IsADirectoryError: If `dst_file` points directly to an existing directory
+    """
+    
+    # Importing here as logger needs various functions from util -> circular imports
+    from .logger import log
+
+    src_file = src_file.resolve()
+    dst_file = dst_file.resolve()
+
+    if dst_file.is_dir():
+        # Cannot overwrite a directory with a file
+        raise IsADirectoryError
+    dst_file.unlink(missing_ok=True)
+    dst_file.parent.mkdir(parents=True, exist_ok=True)
+
+    # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
+    # explicit str() needed for Python 3.8
+    resulting_file = shutil.move(str(src_file), str(dst_file))
+    resulting_file = Path(resulting_file).resolve()
+    log.notice(f"Moved {src_file} to {resulting_file}")
+    return Path(resulting_file)
+    
 
 class DependencyConstraints:
     def __init__(self, base_file_path: Path):

--- a/cibuildwheel/util.py
+++ b/cibuildwheel/util.py
@@ -381,7 +381,6 @@ def move_file(src_file: Path, dst_file: Path) -> Path:
     dst_file = dst_file.resolve()
 
     if dst_file.is_dir():
-        # Cannot overwrite a directory with a file
         msg = "dst_file must be a valid target filename, not an existing directory."
         raise IsADirectoryError(msg)
     dst_file.unlink(missing_ok=True)

--- a/cibuildwheel/util.py
+++ b/cibuildwheel/util.py
@@ -362,7 +362,7 @@ def extract_tar(tar_src: Path, dest: Path) -> None:
 
 def move_file(src_file: Path, dst_file: Path) -> Path:
     """Moves a file safely while avoiding potential semantic confusion:
-     
+
     1. `dst_file` must point to the target filename, not a directory
     2. `dst_file` will be overwritten if it already exists
     3. any missing parent directories will be created
@@ -373,7 +373,7 @@ def move_file(src_file: Path, dst_file: Path) -> Path:
         NotADirectoryError: If any part of the intermediate path to `dst_file` is an existing file
         IsADirectoryError: If `dst_file` points directly to an existing directory
     """
-    
+
     # Importing here as logger needs various functions from util -> circular imports
     from .logger import log
 
@@ -392,7 +392,7 @@ def move_file(src_file: Path, dst_file: Path) -> Path:
     resulting_file = Path(resulting_file).resolve()
     log.notice(f"Moved {src_file} to {resulting_file}")
     return Path(resulting_file)
-    
+
 
 class DependencyConstraints:
     def __init__(self, base_file_path: Path):

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -7,7 +7,6 @@ import subprocess
 import sys
 import textwrap
 from collections.abc import MutableMapping, Sequence, Set
-from contextlib import suppress
 from dataclasses import dataclass
 from functools import lru_cache
 from pathlib import Path

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -542,7 +542,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (remove if already exists)
             if compatible_wheel is None:
-                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
+                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -543,8 +543,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (remove if already exists)
             if compatible_wheel is None:
-                with suppress(FileNotFoundError):
-                    (build_options.output_dir / repaired_wheel.name).unlink()
+                (build_options.output_dir / repaired_wheel.name).unlink(missing_ok=True)
 
                 shutil.move(str(repaired_wheel), build_options.output_dir)
                 built_wheels.append(build_options.output_dir / repaired_wheel.name)

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -546,13 +546,13 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
                 output_wheel.unlink(missing_ok=True)
 
-                # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
+                # shutil.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
                 output_dir.mkdir(parents=True, exist_ok=True)
 
-                # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
+                # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
                 shutil.move(
                     str(repaired_wheel), str(output_wheel)
-                )  # explicit str() needed for mypy3.8
+                )  # explicit str() needed for Python 3.8 - can change to Paths when we drop 3.8 support
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -542,10 +542,16 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (remove if already exists)
             if compatible_wheel is None:
-                build_options.output_dir.joinpath(repaired_wheel.name).unlink(missing_ok=True)
+                output_dir = build_options.output_dir.resolve()
+                output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
+                output_wheel.unlink(missing_ok=True)
 
-                shutil.move(str(repaired_wheel), build_options.output_dir)
-                built_wheels.append(build_options.output_dir / repaired_wheel.name)
+                # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
+                output_dir.mkdir(parents=True, exist_ok=True)
+                
+                # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
+                shutil.move(repaired_wheel, output_wheel)
+                built_wheels.append(output_wheel)
 
             # clean up
             # (we ignore errors because occasionally Windows fails to unlink a file and we

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -550,9 +550,8 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(
-                    str(repaired_wheel), str(output_wheel)
-                )  # explicit str() needed for Python 3.8 - can change to Paths when we drop 3.8 support
+                # explicit str() needed for Python 3.8
+                shutil.move(str(repaired_wheel), str(output_wheel))
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -548,7 +548,7 @@ def build(options: Options, tmp_path: Path) -> None:
 
                 # os.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
                 output_dir.mkdir(parents=True, exist_ok=True)
-                
+
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
                 shutil.move(repaired_wheel, output_wheel)
                 built_wheels.append(output_wheel)

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -546,7 +546,9 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
                 moved_wheel = move_file(repaired_wheel, output_wheel)
                 if moved_wheel != output_wheel.resolve():
-                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
+                    log.warning(
+                        "{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}"
+                    )
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -550,7 +550,7 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(repaired_wheel, output_wheel)
+                shutil.move(str(repaired_wheel), str(output_wheel)) # explicit str() needed for mypy3.8
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -33,6 +33,7 @@ from .util import (
     find_compatible_wheel,
     get_build_verbosity_extra_flags,
     get_pip_version,
+    move_file,
     prepare_command,
     read_python_configs,
     shell,
@@ -542,16 +543,10 @@ def build(options: Options, tmp_path: Path) -> None:
 
             # we're all done here; move it to output (remove if already exists)
             if compatible_wheel is None:
-                output_dir = build_options.output_dir.resolve()
-                output_wheel = output_dir.joinpath(repaired_wheel.name).resolve()
-                output_wheel.unlink(missing_ok=True)
-
-                # shutil.move() will rename the file to what we were expecting to be the parent directory if we don't ensure it exists
-                output_dir.mkdir(parents=True, exist_ok=True)
-
-                # using shutil.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                # explicit str() needed for Python 3.8
-                shutil.move(str(repaired_wheel), str(output_wheel))
+                output_wheel = build_options.output_dir.joinpath(repaired_wheel.name)
+                moved_wheel = move_file(repaired_wheel, output_wheel)
+                if moved_wheel != output_wheel.resolve():
+                    log.warning("{repaired_wheel} was moved to {moved_wheel} instead of {output_wheel}")
                 built_wheels.append(output_wheel)
 
             # clean up

--- a/cibuildwheel/windows.py
+++ b/cibuildwheel/windows.py
@@ -550,7 +550,9 @@ def build(options: Options, tmp_path: Path) -> None:
                 output_dir.mkdir(parents=True, exist_ok=True)
 
                 # using os.move() as Path.rename() is not guaranteed to work across filesystem boundaries
-                shutil.move(str(repaired_wheel), str(output_wheel)) # explicit str() needed for mypy3.8
+                shutil.move(
+                    str(repaired_wheel), str(output_wheel)
+                )  # explicit str() needed for mypy3.8
                 built_wheels.append(output_wheel)
 
             # clean up

